src/shogun/multiclass/tree/CHAIDTree.cpp

```diff
@@ -1392,14 +1392,14 @@ void CHAIDTree::init()
 	m_cont_breakpoints=SGMatrix<float64_t>();
 	m_num_breakpoints=0;
 
-	SG_ADD(&m_weights,"m_weights", "weights", ParameterProperties::READONLY);
-	SG_ADD(&m_weights_set,"m_weights_set", "weights set", ParameterProperties::READONLY);
-	SG_ADD(&m_feature_types,"m_feature_types", "feature types", ParameterProperties::SETTING);
-	SG_ADD(&m_dependent_vartype,"m_dependent_vartype", "dependent variable type", ParameterProperties::SETTING);
-	SG_ADD(&m_max_tree_depth,"m_max_tree_depth", "max tree depth", ParameterProperties::HYPER);
-	SG_ADD(&m_min_node_size,"m_min_node_size", "min node size", ParameterProperties::SETTING);
-	SG_ADD(&m_alpha_merge,"m_alpha_merge", "alpha-merge", ParameterProperties::HYPER);
-	SG_ADD(&m_alpha_split,"m_alpha_split", "alpha-split", ParameterProperties::HYPER);
-	SG_ADD(&m_cont_breakpoints,"m_cont_breakpoints", "breakpoints in continuous attributes", ParameterProperties::SETTING);
-	SG_ADD(&m_num_breakpoints,"m_num_breakpoints", "number of breakpoints", ParameterProperties::HYPER);
+	SG_ADD(&m_weights,"weights", "weights", ParameterProperties::READONLY);
+	SG_ADD(&m_weights_set,"weights_set", "weights set", ParameterProperties::READONLY);
+	SG_ADD(&m_feature_types,"feature_types", "feature types", ParameterProperties::SETTING);
+	SG_ADD(&m_dependent_vartype,"dependent_vartype", "dependent variable type", ParameterProperties::SETTING);
```

Comment:
This causes the notebooks to fail:
https://dev.azure.com/shogunml/shogun/_build/results?buildId=3629&view=logs&j=089c709a-44eb-5f6e-96e7-15e9ee1ff5bf&t=2da3e16b-a2b2-5f01-2cbe-a20d9528195b&l=1849

Should be simple to fix: open the notebook and edit the name in there as well :)

Reply:
How do I run test the notebooks on my local machine? Does `make test` do that?

Reply:
There is a script for doing that https://github.com/shogun-toolbox/shogun/blob/develop/scripts/test_notebooks.sh.

The link Heiko pasted here above will show also how to use it.

Reply:
It doesnt. You would have to compile shogun with the python interface, make sure you can load it from python, and then open the notebook in jupyter notebook.

However, you might be able to do a simple hack here:
1. Open the notebook in a texteditor
2. Search for `m_`
3. If it is one of the varnames, change it to the values you updated them to
4. Save the file in the texteditor and submit

As this is such a simple change, that should do it without the need for you running it locally

Reply:
Yes, I have already pushed the code after making those changes. :)